config/int.go

```diff
@@ -0,0 +1,96 @@
+package config
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// GetInt returns the int representation of the value associated with k.
+//
+// If k is undefined, ok is false and err is nil.
+//
+// If k is defined but its value can not be parsed as an int, err is a non-nil
```

Comment:
```suggestion
// If k is defined but its value cannot be parsed as an int, err is a non-nil
```

Reply:
I also saw multiple `can not`s in the comments and test descriptions.

Reply:
:+1: I always do this.  Both are accepted but `cannot` is definitely more common.  I'll fix these all up in one hit at the end of these PRs.

Reply:
Actually, will do now.